app/content/filters/event.py

```diff
@@ -20,3 +23,13 @@ def filter_expired(self, queryset, name, value):
         if value:
             return queryset.filter(end_date__lt=time).order_by("-start_date")
         return queryset.filter(end_date__gte=time).order_by("start_date")
+
+    def filter_is_admin(self, queryset, name, value):
```

Comment:
Er ikke så fan av å filtrere på admins vie et queryparam. Er det ikke like greit å ha denne i `viewset.get_queryset()`?

Reply:
Letteste måten jeg ser uten å lage et nytt endepunkt a la `api/v1/events/admin/` ettersom man må kunne hente ut alle arrangementer som ikke admin på `tihlde.org/arrangementer/`, men hente ut kun der du er admin på `tihlde.org/admin/arrangementer/`

Reply:
Men er ikke den sjekken like så greit å ha i `viewset.get_queryset()` så admins alltid kan se alle de skal og vanlige medlemmer bare ser vanlige arrangementer?

Reply:
Men vanlige medlemmer skal ha tilgang til å hente ut **alle** arrangementer på samme måte som de som er admin også skal kunne hente ut alle arrangementer. `is_admin` brukes kun for å kunne vise kun arrangementer du har rettighet til å administrere på arrangements-admin-siden. Det er det eneste stedet der det er relevant å ikke hente ut alle arrangementer

Reply:
Da stemmer jeg for å legge det under `admin/arrangementer` så har vi et tydeligere skille på det